docs/semgrep-ci/sample-ci-configs.md

```diff
@@ -311,8 +310,18 @@ These steps can be performed from within Buildkite's interface. From Buildkite's
 
 <TabItem value='buildkite-semgrep'>
 
+The following configuration creates a CI job that runs scans according to the products you have enabled in Semgrep AppSec Platform. The provided environment variables are commonly needed to correctly configure scans from Buildkite.
 
-The following configuration creates a CI job that runs scans depending on what products you have enabled in Semgrep AppSec Platform.
+This file configures two alternative command paths, one for full scans, and one for diff-aware scans. The latter is used for pull or merge requests.
+
+In order for this configuration to run the correct type of scan, go to the pipeline **Settings** and select the connected source code manager in the left sidebar. In that section, make the following changes:
```

Comment:
These steps need to be done in the Buildkite UI, right?

Reply:
yes, I should note that since the rest of it does!